apps/web/components/Deploy/SelfHosted/NodeConfig.tsx

```diff
@@ -39,8 +39,19 @@ const inputBoxDeploymentBlockNumber: Record<number, number> = {
     [sepolia.id]: sepoliaInputBox.receipt.blockNumber,
 };
 
+const epochLengths: Record<number, number> = {
+    [arbitrum.id]: 43200, // arbitrum doesn't have a fixed block interval
+    [arbitrumSepolia.id]: 43200, // arbitrum doesn't have a fixed block interval
+    [base.id]: 43200, // 1 day on a 2s block time
+    [baseSepolia.id]: 43200, // 1 day on a 2s block time
+    [foundry.id]: 720, // 1 hour on a 5s block time
+    [mainnet.id]: 7200, // 1 day on a 12s block time
+    [optimism.id]: 43200, // 1 day on a 2s block time
+    [optimismSepolia.id]: 43200, // 1 day on a 2s block time
+    [sepolia.id]: 7200, // 1 day on a 12s block time
```

Comment:
Is there any expectations on 1 day for testnets 1 week for mainnet? Or this is completely up to the application developer?

Reply:
Indeed. Maybe we could already hint 1 week default for mainnets.

Reply:
Changed to 7 days on mainnets

Reply:
I suggest adding these values somewhere in the docs, not just as code comments.

At apps/docs/guide/running/running-application.md it's said that the default is `7200` but this value will be different according to the chain selected.

Reply:
Running application is about Cartesi CLI.
Maybe somewhere under deployment

Reply:
Maybe I can emphasize that I'm talking about the local CLI env in that section.
And then add this new information at https://docs.sunodo.io/guide/deploying/supported-networks

Reply:
Done